src/compiler/scala/tools/nsc/plugins/Plugin.scala

```
@@ -113,11 +114,14 @@ object Plugin {
   /** Use a class loader to load the plugin class.
    */
   def load(classname: String, loader: ClassLoader): Try[AnyClass] = {
-    Try[AnyClass] {
-      loader loadClass classname
-    } recoverWith {
-      case _: Exception =>
-        Failure(new RuntimeException(s"Warning: class not found: ${classname}"))
+    import scala.util.control.NonFatal
+    try {
+      Success[AnyClass](loader loadClass classname)
+    } catch {
+      case NonFatal(e) =>
+        Failure(new PluginLoadException(classname, s"Error: unable to load class: $classname"))
```

I was responsible for the previous iteration, too.  I don't remember before that (fail silently?) but I do remember wondering if people want to see a stack trace about a missing class, that says "ClassNotFound".  By people, I mean the guy who takes up support for continuations plugin and can't get the class path right in sbt, or whatever.  In Plugins, it recover[Any]s and just uses the message, which would include the causative message IIUC.